### DISCUSSION TITLE
Add possibility to automatically copy TOTP to clipboard

### DISCUSSION THREAD

**Discussion by TahirJalilov** [keepassxreboot/keepassxc#7314] [Ideas]
Make it possible to automatically copy TOTP to clipboard

**Comment by Art4** [comment]
Does [Ctrl]+[T] not work for you?

**Reply by TahirJalilov** [reply]
It works, but I mean to auto copy TOTP after credentials filled

**Comment by droidmonkey** [comment]
We will never implement auto copy to clipboard, it is not a secure way to handle data and we require user interaction to initiate that.

**Reply by TahirJalilov** [reply]
Well, maybe you are right )))
